extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -67,6 +73,8 @@ define('AuthModal', ['jquery', 'wikia.window'], function ($, window) {
 			$blackout.remove();
 			isOpen = false;
 		}
+
+		$(window.document).add(window).off('.authModal');
```

Comment:
You're using document as window property and on line 28 as a global. Let's inject 'wikia.document' into this AMD module and then reference it in a consistent way :)